cmd/virt-handler/virt-handler.go

```diff
@@ -399,6 +400,11 @@ func (app *virtHandlerApp) Run() {
 	go gracefulShutdownInformer.Run(stop)
 	go domainSharedInformer.Run(stop)
 
+	prSockDir, err := safepath.JoinAndResolveWithRelativeRoot("/", reservation.GetPrHelperSocketDir())
```

Comment:
(I am ok if you resolve this comment in follow up). 
I believe we don't need to do this here. I would move the relabeling into context of virt-launcher. Meaning, the virt-launcher is running and before first `sync` is send we can adjust the provided socket to the targeted SELinux context.

Reply:
I did it here because we are labeling at this point also the devices: https://github.com/kubevirt/kubevirt/blob/main/cmd/virt-handler/virt-handler.go#L404-L419